Gemfile

```diff
@@ -30,6 +30,8 @@ gem 'pmp', '~> 0.5'
 
 gem 'hyperresource'
 
+gem 'addressable'
```

Comment:
It is literally not possible.
On Jan 26, 2015 11:22 AM, "Andrew Kuklewicz" <notifications@github.com>
wrote:

> In Gemfile
> <https://github.com/PRX/pmp-importer/pull/9#discussion_r23542047>:
>
> > @@ -30,6 +30,8 @@ gem 'pmp', '~> 0.5'
> >
> >  gem 'hyperresource'
> >
> > +gem 'addressable'
>
> I find it hard to imagine a feed relying on a fragment, but I guess
> anything is possible.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/PRX/pmp-importer/pull/9/files#r23542047>.
>